resources/assets/js/stories/H2Components/Button.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { withIntl } from "storybook-addon-intl";
+import Button from "../../components/H2Components/Button";
+
+const stories = storiesOf("H2 Components/Button", module).addDecorator(
+  withIntl,
+);
+
+stories.add(
+  "Button",
+  (): React.ReactElement => (
+    <section>
+      <Button
+        styling="theme-1, round, medium, solid"
+        data-h2-margin="b(right, 1)"
+      >
```

Comment:
I mentioned elsewhere I'm not sure we need a custom Button component. But even if we don't, its nice to have stories like as examples of how to style them!